.github/workflows/nightly.yml

```diff
@@ -7,10 +7,13 @@
 ---
 name: Pulp Nightly CI/CD
 on:
-  schedule:
-    # * is a special character in YAML so you have to quote this string
-    # runs at 23:00 daily
-    - cron: '00 23 * * *'
+  workflow_run:
+    workflows:
+      - "Pulp on Push"
+    branches:
+      - '*'
```

Comment:
So this means it will run on all branches after something has been pushed?
First of all, that means it's not "nightly" anymore.
Second: Do we want to publish the "nightly" bindings / docs on anything but master?

Reply:
we didn't have any workflow for `push`, on travis, push and cron would run the same "workflow".
I agree this is not nightly anymore, and I don't know if we should change the workflow name, or add another workflow.
For the second question, I was thinking about Y-branches, but now I realize it would release them twice

Reply:
I was thinking we'd have two separate workflows (one for push and one for nightly) since we don't want to publish stuff like the bindings when changes are merged to master?